build.gradle

```diff
@@ -52,9 +52,17 @@ repositories {
         name 'K-4u maven'
         url "https://maven.k-4u.nl/"
     }
-    maven {
+    maven { // backup maven location
         name "Tehnut Maven FS"
-        url "https://tehnut.info/maven"
+        url "https://github.com/TehNut/temporary-maven-thing/tree/master/maven/"
+    }
+    maven { // location of the maven that hosts JEI files
+        name = "Progwml6 maven"
+        url = "https://dvs1.progwml6.com/files/maven/"
+    }
```

Comment:
Progwml6's maven is already in the build.gradle as 'DVS1 Maven FS', please remove the duplicate.

Reply:
Done.